### ISSUE TITLE
an object with bytes must have text content

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#77]
This is not a valid XMIR:

```
<o base="org.eolang.string" name="foo">
  <o base="org.eolang.bytes"/>
</o>
```

An object with `base="org.eolang.bytes"` must have `text()` content. Let's create a new lint to check for this.

**Comment by h1alexbel** [comment]
@yegor256 I can work on this

**Comment by maxonfjvipon** [comment]
@yegor256 I think it's a valid XMIR. Consider the next code.
EO:
```
string bytes > x
```
XMIR:
```xml
<o base="string" name="x">
  <o base="bytes"/>
</o>
```

**Comment by yegor256** [comment]
@maxonfjvipon I would make it an error. The right way to do it is this:

```
string -- > x
```

**Comment by maxonfjvipon** [comment]
@yegor256 so you suggest to prohibit explicit usage of `bytes`?

Since recently `bytes` has one free attribute which should (but not must) be an abstract with \Delta. 

But the next case looks valid:
```
string
  bytes
    []
      01- > @
```

Yes, 99% will never write like this, but it's valid and dataizable

**Comment by yegor256** [comment]
@maxonfjvipon I suggest we fix the XMIR, to make it aligned with phi

**Comment by maxonfjvipon** [comment]
@yegor256 @h1alexbel the XMIR is fixed, so you can continue to work on the issue
